crates/loam-cli/src/commands/build/env_toml.rs

```diff
@@ -0,0 +1,68 @@
+use std::collections::BTreeMap as Map;
+use std::io;
+use std::ops::Deref;
+use std::path::PathBuf;
+
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
+    #[error("⛔ ️parsing environments.toml: {0}")]
+    ParsingToml(io::Error),
+    #[error("⛔ ️no settings for current LOAM_ENV ({0:?}) found in environments.toml")]
+    NoSettingsForCurrentEnv(String),
+}
+
+#[derive(Debug, serde::Deserialize)]
+struct Environments(Map<Box<str>, Environment>);
+
+impl Deref for Environments {
+    type Target = Map<Box<str>, Environment>;
+
+    fn deref(&self) -> &Self::Target {
+        &self.0
+    }
+}
+
+#[derive(Debug, serde::Deserialize, Clone)]
+pub struct Environment {
+    pub accounts: Option<Vec<Account>>,
+    pub network: Network,
+    pub contracts: Option<Map<Box<str>, Contract>>,
+}
+
+#[derive(Debug, serde::Deserialize, Clone)]
+#[serde(rename_all = "kebab-case")]
+pub struct Network {
+    pub name: Option<String>,
+    pub rpc_url: Option<String>,
+    pub network_passphrase: Option<String>,
+    // run_locally: Option<bool>,
+}
+
+#[derive(Debug, serde::Deserialize, Clone)]
+pub struct Account {
+    pub name: String,
+    pub default: Option<bool>,
+}
+
+#[derive(Debug, serde::Deserialize, Clone)]
+pub struct Contract {
+    pub workspace: Option<bool>,
+}
+
+impl Environment {
+    pub fn get(workspace_root: &PathBuf, loam_env: String) -> Result<Option<Environment>, Error> {
```

Comment:
```suggestion
    pub fn get(workspace_root: &Path, loam_env: &str) -> Result<Option<Environment>, Error> {
```

Reply:
oh, I see, you were suggesting combining this with the above suggestion. sure.